app/controllers/subscription.js

```diff
@@ -205,13 +207,18 @@ export default Ember.Controller.extend({
       return;
     }
 
-    if (["create","update"].contains(data.operation)) {
+    if (["create","update"].includes(data.operation)) {
         var payload = {};
         payload[type] = item;
         this.store.pushPayload(payload);
     } else if (existingItem) { //delete
       this.store.unloadRecord(existingItem);
     }
     run(success);
+
+    // mark message as read if message will appear in current view
+    if (type === "message") {
+      console.log('mark messages read here');
```

Comment:
Something missing ? :)

Reply:
yes @patrixr , mark read feature in this chat is pending, I'll remove this code for now.